sparkey/__init__.py

```diff
@@ -122,9 +124,16 @@ def wrapper(*a, **kw):
 
 if str == bytes:
     def _to_bytes(s, name):
-        if type(s) != str:
-            raise SparkeyException(s + " must be a string")
+        if type(s) != str and type(s) != future.types.newstr:
+            raise SparkeyException(name + " must be a string")
         return s
+    
+    def _to_str(b, name):
+        if b is None: return None
+        if type(b) != str:
```

Comment:
should this also compare with newstr?

Reply:
newstr is just for python2 back compatibility, since we're in the py3 block, shouldn't need ot